src/canvas/SelectableCanvas.ts

```diff
@@ -680,31 +684,46 @@ export class SelectableCanvas<
           target.cacheTranslationY + normalizedPointer.y * target.zoomY,
           0
         );
+      // transform tolerance according to vpt
+      // TODO: use sendVectorToPlane
+      const tolerance = new Point()
+        .scalarAdd(this.targetFindTolerance)
+        .transform(invertTransform(this.viewportTransform), true);
+      const size = tolerance.scalarMultiply(2).max(new Point(1, 1));
```

Comment:
i need to think about this. this put ourselves in the case of transforming the tolerance with the vpt when what we want is really just canvas pixels. So if there is a small text or a thin line and the zoom level makes it super thin, we were not composensating for this, if the line was so thinned by the zoom out that was nearly transparent, we would consider it transparent before.

Reply:
@shaman can you share the app code used to test this so i can reproduce the testing case? i m curious about trying a couple of things.

Reply:
The method accepts a global pointer, living in the viewport, so 1 is the what is perceived as 1 by the user, not by the canvas.

Reply:
yes 1 viewport pixel. 1 of what are you looking at in that moment and also 1 of how much you can move your mouse.
The need to use the max between scalarMultiply and 1, and the invert of the viewportTransform is something you added, i think it change the meaning of what tolerance does.

Reply:
I think I fixed it because previously it was checking tolerance in the canvas plane and not in the viewport. I will add a dedicated test.

Reply:
The app... I will share a link?

Reply:
i made mine, i had this time to work on it and i just played a bit
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/1194048/224551400-854d3931-a9f1-42b8-a00d-49200f8e37e8.png">

I made this canvas where i can see what is in the pixelFind canvas in realtime, as soon as i m done highlighting the difference i can push up the branch